tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -1421,6 +1421,14 @@ def padded_batch(self,
     * If the dimension is unknown, the component will be padded out to the
       maximum length of all elements in that dimension.
 
+    The `padding_values` argument determines the values used to pad each
+    component to the respective shape. The `padding_values` should have the
+    same structure as the input dataset. If `padding_values` is a single
+    element and the input dataset has multiple components, then the same
+    `padding_values` will be used to pad every component of the dataset.
+    If `padding_values` is a scalar, then its value will be broadcasted
+    to match the shape of each component.
```

Comment:
This should be included in the "Args" section of the docstring instead.

Reply:
@jsimsa Thanks. Updated.